pocket_coffea/lib/weights_manager.py

```diff
@@ -229,7 +230,7 @@ def _compute_weight(self, weight_name, events, shape_variation):
         in the constructor.
         '''
         if weight_name == "genWeight":
-            return [('genWeight', events.genWeight)]
+            return [('genWeight', events.genWeight / self._sum_genweights)]
```

Comment:
Each event is weighted by the `genWeight` divided by the sum of the `genWeights`.